package.json

```diff
@@ -11,94 +11,80 @@
     "build:cordova": "IS_CORDOVA=true npm run build",
     "build:ios": "npm run build:cordova && cordova build ios",
     "build:android": "npm run build:cordova && cordova build android",
-    "gen:cordova-resources": "cordova-res",
+    "gen:cordova-resources": "cordova-res --icon-background-source '#f7296e'",
     "lint": "NODE_ENV=production vue-cli-service lint --no-fix",
     "test": "npm run lint && npm run test:unit && npm run test:e2e -- --headless",
-    "storybook:build": "vue-cli-service storybook:build -c config/storybook",
-    "storybook:serve": "vue-cli-service storybook:serve -p 6006 -c config/storybook",
     "test:e2e": "vue-cli-service test:e2e",
     "test:unit": "vue-cli-service test:unit",
     "i18n:report": "vue-i18n-extract report --vueFiles './src/**/*.?(js|vue)' --languageFiles './src/locales/**/*.json'"
   },
   "dependencies": {
-    "@aeternity/aepp-sdk": "^8.1.0",
+    "@aeternity/aepp-sdk": "github:aeternity/aepp-sdk-js#develop",
```

Comment:
I don't like an idea to depend from the develop branch.
Also the installed package from the develop branch doesnt have a lot of prebuild files inside `es` folder.

Reply:
this is not compatible with Windows, but it is the fastest way to make a release of Base aepp 🙃